# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../wv2/gcc43.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- wv2-0.2.3/src/styles.cpp.vanilla	2008-04-09 14:52:34.000000000 +0000
+++ wv2-0.2.3/src/styles.cpp	2008-04-09 14:52:46.000000000 +0000
@@ -23,6 +23,8 @@
 #include "paragraphproperties.h"
 
 #include "wvlog.h"
+
+#include <string.h>
 #include <algorithm>
 
 namespace wvWare
